### PR TITLE
[**Part-2**] Neo module/master fixes

### PR DESCRIPTION
**Pull request by cschuchardt88**
# Change Log
- Fixed `config.json`
- Renamed `config.json` to `$PluginName.json`

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by vncoelho — DISMISSED** [review]
It was initially tested with this PR.

File: src/Plugins/OracleService/OracleService.cs [hunk 1]
```diff
@@ -54,12 +54,14 @@ public class OracleService : Plugin
         private OracleStatus status = OracleStatus.Unstarted;
         private IWalletProvider walletProvider;
         private int counter;
-        private NeoSystem System;
+        private NeoSystem system;
```
Comment (shargon) [inline src/Plugins/OracleService/OracleService.cs:57]:
```suggestion
        private NeoSystem _system;
```

File: src/Plugins/StateService/StatePlugin.cs [hunk 2]
```diff
@@ -37,11 +37,12 @@ public class StatePlugin : Plugin
         public const string StatePayloadCategory = "StateService";
         public override string Name => "StateService";
         public override string Description => "Enables MPT for the node";
+        public override string ConfigFile => System.IO.Path.Combine(RootPath, "StateService.json");
 
         internal IActorRef Store;
         internal IActorRef Verifier;
 
-        internal static NeoSystem System;
+        internal static NeoSystem system;
```
Comment (shargon) [inline src/Plugins/StateService/StatePlugin.cs:45]:
```suggestion
        internal static NeoSystem _system;
```

**Review by AnnaShaleva — COMMENTED** [review]
I'm wondering about the following fact: new `PluginName.json` configuration files are added in this PR, but you haven't removed the corresponding old `config.json` configuration files.

**Comment by vncoelho** [comment]
> I'm wondering about the following fact: new `PluginName.json` configuration files are added in this PR, but you haven't removed the corresponding old `config.json` configuration files.

Old config are for neo-cli now

**Comment by cschuchardt88** [comment]
> > I'm wondering about the following fact: new `PluginName.json` configuration files are added in this PR, but you haven't removed the corresponding old `config.json` configuration files.
> 
> Old config are for neo-cli now

They needed to be deleted, Not `neo-cli` but the plugin old `config.json` files

**Comment by vncoelho** [comment]
> I'm wondering about the following fact: new `PluginName.json` configuration files are added in this PR, but you haven't removed the corresponding old `config.json` configuration files.

Now I  saw that the files were there...ahahah

**Review by Jim8y — APPROVED** [review]
No major changes. LGTM

**Review by vncoelho — APPROVED** [review]

**Comment by vncoelho** [comment]
@NGDAdmin , this is urgent. Currently master is broken without this.

**Review by shargon — APPROVED** [review]

**Review by vncoelho — APPROVED** [review]

**Pull request merged**